trlx/model/accelerate_base_model.py

```diff
@@ -210,6 +213,7 @@ def evaluate(self):
                     columns_data.append(values)
 
             rows = list(zip(*columns_data))
+            print(rows[0])
```

Comment:
nit: should comment this out

Reply:
I like the functionality of it, both for myself and for when people just make screenshot of their terminal and share how ppo collapses, you can witness it all